src/rpc/methods/chain.rs

```diff
@@ -985,6 +985,22 @@ impl RpcMethod<1> for ChainGetTipSet {
     }
 }
 
+pub enum ChainGetTipSetV2 {}
+impl RpcMethod<1> for ChainGetTipSetV2 {
+    const NAME: &'static str = "Filecoin.ChainGetTipSet";
+    const PARAM_NAMES: [&'static str; 1] = ["tipsetSelector"];
+    const API_PATHS: BitFlags<ApiPaths> = make_bitflags!(ApiPaths::{ V2 });
+    const PERMISSION: Permission = Permission::Read;
+    const DESCRIPTION: Option<&'static str> = Some("Returns the tipset with the specified CID.");
+
+    type Params = (ApiTipsetKey,);
```

Comment:
It should be `TipsetSelector`. The method will be implemented in a subsequent PR.

```go
type TipSetSelector struct {
	Key    *TipSetKey    `json:"key,omitempty"`
	Height *TipSetHeight `json:"height,omitempty"`
	Tag    *TipSetTag    `json:"tag,omitempty"`
}
```